charts/px/templates/hooks/pre-install/etcd-preinstall-hook.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: kube-system
+  name: px-etcd-preinstall-hook
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: pre-install-job
+        terminationMessagePath: '/dev/termination-log'    
+        terminationMessagePolicy: 'FallbackToLogsOnError'        
+        imagePullPolicy: Always
+        image: "hrishi/px-etcd-preinstall-hook:v1"
```

Comment:
this needs to be a portworx repo. Create a issue to fix this so we don't block this PR.

@shailvipx can you create a repo `hrishi/px-etcd-preinstall-hook` under Portworx. Give our usual build user write access.